cloudinit/config/cc_grub_dpkg.py

```diff
@@ -22,9 +22,8 @@
 MODULE_DESCRIPTION = """\
 Configure which device is used as the target for grub installation. This module
 can be enabled/disabled using the ``enabled`` config key in the ``grub_dpkg``
-config dict. The global config key ``grub-dpkg`` is an alias for ``grub_dpkg``.
-If no installation device is specified this module will execute grub-probe to
-determine which disk the /boot directory is associated with.
+config dict. This module automatically selects a disk with ``grub-probe`` if no
```

Comment:
```suggestion
config dict. This module automatically selects a disk using ``grub-probe`` if no
```

Reply:
This read a little ambiguously at first. I think "using" is a more accurate reflection of what's happening (happy to be corrected though if I'm off base)